react/components/Title/index.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+
+import { LEFT, RIGHT, CENTER } from './constants.js'
+import title from './title.css'
+
+/**
+ * Name component. Show name and relevant SKU information of the Product Summary
+ */
+class Title extends Component {
+  static propTypes = {
+    /** Text to be displayed */
+    content: PropTypes.string,
```

Comment:
Why not just render `children` instead of using this prop `content`? 🤔

Reply:
We just want to render a heading (E.g. `<h3>`). If we use `props.children`, how can we make sure that we have only one of this kind of element? We were referring to the `ProductName` component here.